mysql-test/suite/sys_vars/r/innodb_default_encryption_key_id_basic.result

```diff
@@ -1,3 +1,140 @@
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+4294967294
+SET persist innodb_default_encryption_key_id=4294967295;
+Warnings:
+Warning	1292	Truncated incorrect innodb_encryption_key_id value: '4294967295'
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+4294967294
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+4294967294
+SET persist_only innodb_default_encryption_key_id=4294967295;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+4294967294
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+4294967294
+SET innodb_default_encryption_key_id=-1;
+Warnings:
+Warning	1292	Truncated incorrect innodb_encryption_key_id value: '-1'
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+4294967294
+SET global innodb_default_encryption_key_id=-1;
+Warnings:
+Warning	1292	Truncated incorrect innodb_encryption_key_id value: '-1'
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
+SET persist innodb_default_encryption_key_id=-1;
+Warnings:
+Warning	1292	Truncated incorrect innodb_encryption_key_id value: '-1'
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
+SET persist_only innodb_default_encryption_key_id=-1;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
+SET innodb_default_encryption_key_id=0;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
+SET global innodb_default_encryption_key_id=0;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+0
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
+SET innodb_default_encryption_key_id=1;
+SET global innodb_default_encryption_key_id=1;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+1
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+1
+SET persist innodb_default_encryption_key_id=0;
+SELECT @@session.innodb_default_encryption_key_id;
+@@session.innodb_default_encryption_key_id
+1
+SELECT @@global.innodb_default_encryption_key_id;
+@@global.innodb_default_encryption_key_id
+0
```

Comment:
is this expected SET PERSIST behaviour? This changes the global value and not the session value. Can you please check SET PERSIST with another session variable?

Reply:
@satya-bodapati, to my knowledge this is expected - SET PERSIST updates global variable only.

Reply:
ok. Can you also check the Jenkins results. the test is skipped on release builds
201002 11:06:48 [ 77%] sys_vars.innodb_default_encryption_key_id_basic w4  [ skipped ]  Not implemented

Please resubmit to Jenkins (some issue with debug builds)